packages/core/contracts/insured-bridge/BridgePool.sol

```diff
@@ -387,7 +391,6 @@ contract BridgePool is Testable, BridgePoolInterface, ExpandedERC20, MultiCaller
         emit RelaySettled(depositHash, relayHash, msg.sender);
 
         delete relay.realizedLpFeePct;
-        delete relay.instantRelayer;
```

Comment:
Why can't we still delete the instant relayer (to get a refund)? Once it's settled, there should be no way to create a request with this deposit hash anyway, right?

Reply:
true! I forgot about the other mapping